.env.production

```diff
@@ -0,0 +1,7 @@
+NEXT_PUBLIC_ENV=PROD
```

Comment:
I think this file shouldn't be part of the repo.

Ca

Reply:
It would be better to inject it with netlify

Reply:
I would disagree – the contents of .env.development and .env.production are public knowledge anyways, and including them here significantly simplifies onboarding of new engineers and communication in our team. E. g. when the structure changed and one team member is on vacation, it's very hard to continue developing/testing the repository.

Reply:
I don't have a strong opinion about it, if it contains no secrets and it makes it easier for you to develop at the current stage. However, if we move to a paid version of infura our link would be searchable if it is stored public on github. 

Obviously you could look it up if you use the tinlake-ui in the browser but not searchable via web.

Reply:
Fair point, I'll add a comment to the .env file to make sure we don't expose it.

Reply:
If we use the paid infura version we could us a reverse proxy like nginx

Reply:
Absolutely 💯